src/Avalonia.FuncUI/DSL/SelectableTextBlock.fs

```diff
@@ -0,0 +1,20 @@
+namespace Avalonia.FuncUI.DSL
+
+
+[<AutoOpen>]
+module SelectableTextBlock =  
+    open Avalonia.Controls
+    open Avalonia.Controls.Documents
+    open Avalonia.FuncUI.Builder
+    open Avalonia.FuncUI.Types
+
+    let create (attrs: IAttr<SelectableTextBlock> list): IView<SelectableTextBlock> =
+        ViewBuilder.Create<SelectableTextBlock>(attrs)
+    
+    type SelectableTextBlock with
+        static member inlines<'t when 't :> SelectableTextBlock>(value: InlineCollection) : IAttr<'t> =
```

Comment:
Since this appeared to have been moved up to TextBlock, could we move it there as well? Since `SelectableTextBlock` inherits from `TextBlock` it should get it for free as well.

Anyway there seems to be a bunch of changes in the PR you linked, so I'd probably take another look at everything to see that it's up to date. Feel free to close and merge if you prefer to do so, I can do it 😃

Reply:
I've moved the Inlines property into TextBlock.

As far as other changes go, there are some selection related properties on SelectableTextBlock that could have bindings added seperately?

Reply:
Yep yep, we can address that in another PR. Thanks for the work on this one, we can merge it!